build/bin/release

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+if [ "$#" -lt "1" ]; then
+    echo "must specify chart directory"
+    exit 1
+fi
+
+set -x
+
+chart=$1
+
+rm -r build/release
+
+owner=${GITHUB_USER:-onosproject}
+repo=${GITHUB_REPO:-onos-helm-charts}
+remote=${GIT_REMOTE:-origin}
+token=${GITHUB_TOKEN}
+branch=$(git branch --show-current)
+
+# Package the Helm chart
+helm package $chart --destination build/release
+
+# Upload the Helm chart release
+cr upload \
```

Comment:
Will this refuse (fail) to re-release an already released chart?

Reply:
Added `set -e`